src/app/_components/ListboxOptions.tsx

```diff
@@ -2,21 +2,31 @@ import { ListboxOptions as HeadlessUIListboxOptions } from '@headlessui/react'
 import { clsx } from 'clsx'
 
 type ListboxOptionsProps = {
-  position?: 'left' | 'right'
   children: React.ReactNode
+  position?: 'left' | 'right' | 'bottom'
+  inheritWidth?: 'button'
 }
 
 export function ListboxOptions({
-  position = 'left',
   children,
+  position = 'left',
+  inheritWidth,
 }: ListboxOptionsProps) {
-  const positionClass = position === 'right' ? 'right-6 md:right-auto' : ''
+  const positionClass = clsx({
+    'right-6 md:right-auto': position === 'right',
+    'md:bottom-auto': position === 'bottom',
```

Comment:
We should preferably use the new anchor API for positioning, as mentioned in our [specs](https://www.notion.so/filecoin/FF-Refactor-Listbox-12a7631f282580edb479cdcce2deea6f?pvs=4#1357631f2825802ba677ee00a17ff1dd).

[V2 docs](https://headlessui.com/react/listbox#basic-example) for reference.